src/ImageSharp/Formats/Webp/Lossless/Vp8LHistogram.cs

```diff
@@ -505,11 +511,48 @@ private static double ExtraCost(Span<uint> population, int length)
             return cost;
         }
 
-        private static void AddVector(uint[] a, uint[] b, uint[] output, int size)
+        private static void AddVector(Span<uint> a, Span<uint> b, Span<uint> output, int size)
```

Comment:
We should `DebugGuard` for count mismatch.

Nit: `count` would be better name than `size` here.